js/ui/components/clubhouse.js

```diff
@@ -114,12 +115,33 @@ var ClubhouseNotificationSource = new Lang.Class({
     },
 });
 
+var NotifiableButton = new Lang.Class({
+    Name: 'NotifiableButton',
```

Comment:
I think it'd be fine to use this name and the overall approach if we were to have more than one button that's notifiable. Since that's not the case for now, I'd prefer to encapsulate things more and call it really something like `ClubhouseOpenButton`. See the rest of the comments which are related to this.

Reply:
Sounds good. `NotifiableButton` was suggested by you in the previous review pass.